i18n/babel-plugin.js

```diff
@@ -58,7 +58,7 @@ const DEFAULT_FUNCTIONS = {
 	__: [ 'msgid' ],
 	_n: [ 'msgid', 'msgid_plural' ],
 	_x: [ 'msgid', 'msgctxt' ],
-	_nx: [ 'msgid', 'msgctxt', 'msgid_plural' ],
+	_nx: [ 'msgid', 'msgid_plural', '_', 'msgctxt' ],
```

Comment:
Following the logic, I don't think we really need to assign anything for the number argument. Even `null` or `undefined` would suffice:

https://github.com/WordPress/gutenberg/blob/3134e1acd2ff42e3eeb0a245d5d625952521fc29/i18n/babel-plugin.js#L213-L214

https://github.com/WordPress/gutenberg/blob/3134e1acd2ff42e3eeb0a245d5d625952521fc29/i18n/babel-plugin.js#L167-L169

https://github.com/WordPress/gutenberg/blob/3134e1acd2ff42e3eeb0a245d5d625952521fc29/i18n/babel-plugin.js#L76